config/src/main/resources/ropsten.json

```diff
@@ -9,6 +9,7 @@
     "constantinopleBlock": 4230000,
     "constantinopleFixBlock": 4939394,
     "istanbulBlock": 6485846,
+    "icetanbulBlock": 7117117,
```

Comment:
still icestanbul here, is that intentional?

Reply:
Nope, and a test should have caught that too.  Patches incoming.